### DISCUSSION TITLE
Documentation issue

### DISCUSSION THREAD

**Discussion by KyleC69** [AvaloniaUI/Avalonia#12120] [General]
On documentation page 'https://docs.avaloniaui.net/docs/next/concepts/reactiveui/reactive-command' the braces are missing from the property bindings.

**Comment by timunie** [comment]
Please open doc issues at: https://github.com/AvaloniaUI/avalonia-docs 

Thank you for your support 🙏 

Also PRs are welcome to fix wrong or missing information, if you have time to do.
